gamification/management/commands/create_gamification_data.py

```diff
@@ -0,0 +1,20 @@
+from django.core.management.base import BaseCommand
+
+from gamification.create import (
+    create_levels,
+    create_badge_activity,
+    create_badges,
+    add_activity_to_badge,
+    )
+from gamification.newcomers import create_newcomers
+
+
+class Command(BaseCommand):
+    help = 'Create gamification data'
```

Comment:
"data" is too generic as "data" is also used for participant data.

Typically this stuff is "config data", or "reference data"

Also put all of those files into a common subdirectory, like "gamification/init/points.py" . If you can think of an alphabetically earlier name than "init" that would be good.  Maybe "begin".  And then rename other stuff to appear after it, so that patches always include any of this reference data changes first before code changes.

Create other subdirectories as needed to group related modules together.